src/main/cli/settings.c

```diff
@@ -1130,6 +1130,7 @@ const clivalue_t valueTable[] = {
 // PG_OSD_CONFIG
 #ifdef USE_OSD
     { "osd_units",                  VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_UNIT }, PG_OSD_CONFIG, offsetof(osdConfig_t, units) },
+    { "osd_dynamic_distance_units", VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_OSD_CONFIG, offsetof(osdConfig_t, dynamic_distance_units) },
```

Comment:
Not sure why we should make this optimal?

Reply:
Do you mean "optional"?

The current logic simply displays meters or feet and always has. So I'd have to assume that some people probably like this and will complain if changed (as is always the case). I could see an argument to make the new behavior the default though while retaining the option to turn it off and revert to the old style.

Reply:
I see this as a functional improvement. In this respect I am not sure that we should give users the option to retain the old behaviour with its shortcomings, unless there is a use case for it.

Reply:
I don't have a strong opinion one way or another - just trying to prevent the inevitable complaints from some that they liked the way it was before and don't like the change. One of the use cases some of the long-range fanatics do is try to beat their "personal best" based on the post-flight stats. So I can see them complaining that > 10 (with 1 decimal) their accuracy drops to 100m or 528ft depending on the units.

Reply:
Hmm, I kind of think that making this configurable to start with kind of sets the precedent that this should be configurable, prompting users to complain if we remove this capability later - if we don't make it configurable to start with, we can always add that if users come up with good use cases.